### PR TITLE
Hangfire jobs use events time zone

### LINKED ISSUE #1493: Update Hangfire job code to convert to Event's TimeZoneId
@tonysurma please assign this to me.

I am currently working on this.

### PR DESCRIPTION
**Pull request by mgmccarthy**
Fixes #1493

### CODE REVIEW

**Comment by dpaquette** [comment]
I plan to review this tonight

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/InitialRequestConfirmationsSentHandlerShould.cs [hunk 1]
```diff
@@ -19,15 +18,21 @@ public class InitialRequestConfirmationsSentHandlerShould : InMemoryContextTest
         [Fact]
         public async Task UpdateRequestStatusToPendingConfirmation()
         {
+            var dateTimeNow = DateTime.Now;
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/InitialRequestConfirmationsSentHandlerShould.cs]: To match the behavior of Itineraries in the system, this should be DateTime.Today

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/InitialRequestConfirmationsSentHandlerShould.cs [hunk 2]
```diff
@@ -37,26 +42,31 @@ public class InitialRequestConfirmationsSentHandlerShould : InMemoryContextTest
         }
 
         [Fact]
-        public async Task ScheduleISendRequestConfirmationMessagesAWeekBeforeAnItineraryDateWithTheCorrectMethodSignatureAndTheCorrectDate()
+        public async Task ScheduleISendRequestConfirmationMessagesSevenDaysBeforeAnItineraryDateWithTheCorrectMethodSignatureAndTheCorrectDate()
         {
+            var dateTimeNow = DateTime.Now;
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/InitialRequestConfirmationsSentHandlerShould.cs]: DateTime.Today. The Itinerary date represents a date only, no time.

File: AllReadyApp/Web-App/AllReady.UnitTest/Hangfire/Jobs/SendRequestConfirmationMessagesADayBeforeAnItineraryDateShould.cs [hunk 3]
```diff
@@ -63,75 +62,96 @@ public void NotSendRequestConfirmations_WhenRequestIdsMatchExistingRequests_AndT
...
-        public void ScheduleISendRequestConfirmationMessagesTheDayOfAnItineraryDateWithCorrectParameters_WhenRequestIdsMatchExistingRequests_AndThoseRequestsHaveAStatusOfPendingConfirmation_AndTodayIsLessThanOneDayBeforeTheItineraryDate()
+        public void NotSendRequestConfirmations_WhenRequestIdsMatchExistingRequests_AndThoseRequestsHaveAStatusOfPendingConfirmation_AndTodayIsNotOneDayBeforeTheItineraryDate()
         {
-            var dateTimeUtcNow = DateTime.UtcNow;
+            var dateTimeNow = DateTime.Now;
+            var dateTimeNowUnspecified = DateTime.SpecifyKind(dateTimeNow, DateTimeKind.Unspecified);
+            var dateTimeUtcNow = DateTime.SpecifyKind(dateTimeNow, DateTimeKind.Utc);
 
+            var @event = new Event { Id = 1, TimeZoneId = "Eastern Standard Time" };
+            var itinerary = new Itinerary { Id = 1, Date = dateTimeNowUnspecified.Date, EventId = @event.Id, Event = @event };
```
Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady.UnitTest/Hangfire/Jobs/SendRequestConfirmationMessagesADayBeforeAnItineraryDateShould.cs]: Yup, like this whenever you create an itinerary

**Comment by mgmccarthy** [comment]
@dpaquette, when this build completes, can you eyeball this? All `DateTime.Now`'s change to `DateTime.Today`'s

**Comment by tonysurma** [comment]
@mgmccarthy once @dpaquette does a quick review and its good I will merge it in

**Comment by dpaquette** [comment]
:shipit:

**Pull request merged**